R/extendr-wrappers.R

```diff
@@ -1,7 +1,4 @@
 # Generated by extendr: Do not edit by hand
-
-# nolint start
-
```

Comment:
Please use the latest rextendr

Reply:
I tried `rextendr-0.3.1`, but this comments seems to show up as well. Do you mean the latest commit of `rextendr`?

Reply:
The addition of this comment is intentional (rextendr >= 0.3.0); see extendr/rextendr#214.